__tests__/app.test.js

```diff
@@ -289,4 +280,72 @@ describe('POST "/api/articles/:article_id/comments"', () => {
       expect(response.body.msg).toBe('not found')
     })
   })
+});
+
+describe('PATCH /api/articles/:article_id ', () => {
+  test('should return updated article object with correct votes value', () => {
+    const newVote = 1
+    const updateVotes = {inc_votes: newVote}
+    return request(app)
+    .patch("/api/articles/1")
+    .send(updateVotes)
+    .expect(200)
+    .then((response) => {
+      const article = response.body.article
+      expect(article.article_id).toBe(1)
+      expect(article.votes).toBe(101)
+    })
+  });
+
+  test('should decrement votes value when newVote value is negative', () => {
+    const newVote = -30
+    const updateVotes = {inc_votes: newVote}
+    return request(app)
+    .patch("/api/articles/1")
+    .send(updateVotes)
+    .expect(200)
+    .then((response) => {
+      const article = response.body.article
+      expect(article.article_id).toBe(1)
+      expect(article.votes).toBe(70)
+    })
+  });
+
+  test('returns 400 bad request if there is no patch body', () => {
+    const updateVotes = {}
+    return request(app)
+    .patch("/api/articles/1")
+    .send(updateVotes)
+    .expect(400)
+    .then((response) => {
+      expect(response.body.msg).toBe('bad request')
+      
+    })
+  })
+
+  test('returns 404 if article does not exist', () => {
+    const newVote = -30
+    const updateVotes = {inc_votes: newVote}
+    return request(app)
+    .patch("/api/articles/999")
+    .send(updateVotes)
+    .expect(404)
+    .then((response) => {
+      expect(response.body.msg).toBe('not found')
+      
+    })
+  })
+
+  test('returns 400 if votes increment is invalid', () => {
+    const newVote = 'not-a-number'
+    const updateVotes = {inc_votes: newVote}
+    return request(app)
+    .patch("/api/articles/1")
+    .send(updateVotes)
+    .expect(400)
+    .then((response) => {
+      expect(response.body.msg).toBe('bad request')
+      
+    })
```

Comment:
Great start here, what about updating votes on an article_id of banana? What would/should happen?